satellite/server_register.yml

```diff
@@ -2,7 +2,7 @@
 - hosts: "{{ HOSTS }}"
   become: yes
   vars:
-    env: undef
+    # env: undef
```

Comment:
Why comment this out?

Reply:
This is in preparation for the 3 nodes when we get that arch  :-)